ncrypt/crypt.c

```diff
@@ -1209,6 +1209,12 @@ int mutt_protected_headers_handler(struct Body *b_email, struct State *state)
 
   if (c_devel_security)
   {
+    if (b_email->mime_headers->message_id &&
+        (!display || !c_weed || !mutt_matches_ignore("message-id")))
```

Comment:
Hmmm, I just noticed that I should write `!weed` for consistency with the rest.  We probably changed that after I wrote this patch.

Reply:
Done.